src/Microsoft.Health.Fhir.Azure/ExportDestinationClient/AzureExportDestinationClient.cs

```diff
@@ -48,12 +49,12 @@ public async Task ConnectAsync(string connectionSettings, CancellationToken canc
             }
             catch (Exception)
             {
-                throw new DestinationConnectionException(Resources.InvalidConnectionSettings);
+                throw new DestinationConnectionException(Resources.InvalidConnectionSettings, HttpStatusCode.BadRequest);
```

Comment:
We should consider validating this during queuing since there is no point queuing the job if the input is invalid.

Reply:
Added a check to see whether string is base64 encoded at the filter attribute layer.